tools/cert-sidecar/app/x509IdentityMgmt/Requests.js

```diff
@@ -133,6 +134,42 @@ class Requests {
       throw new Error('Cannot retrieve CA certificate');
     }
   }
+
+  /**
+   * Obtains a CA's certificate bundle trusted by the platform.
+   * The first certificate in the bundle is that of the platform's CA,
+   * the rest are external CAs that have been registered as trusted.
+   *
+   * @throws Will throw an error if cannot retrieve  CA certificate
+   *
+   * @returns {String|null} PEM encoded Certificate
+   */
+  async getCACertBundle() {
+    this.logger.debug("getCABundle: Getting the trusted CA's certificate bundle...");
+    try {
+      const {
+        status,
+        statusText,
+        data,
+      } = await this.axiosX509.get(this.paths.caBundle, {
+        headers: {
+          Accept: 'application/x-pem-file',
```

Comment:
Can this accept be used on the other endpoints or just this one? Is it something new?

Reply:
It is new, it is not implemented in the other endpoints, but we can make this improvement! :+1:

Reply:
PR submitted for approval:
https://github.com/dojot/dojot/pull/2052

Reply:
I applied the changes in this PR to support the changes made in PR https://github.com/dojot/dojot/pull/2052